Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:18-alpine AS development
+
+WORKDIR /app
+
+COPY package*.json /app/
+
+RUN npm ci
+
+COPY . .
+
```

Comment:
Add a command: `NODE_BUILD=true npm run build`
You can setup the environment variable `NODE_BUILD=true` on a separate line if necessary.